include/nlohmann/detail/conversions/from_json.hpp

```diff
@@ -448,6 +448,12 @@ std::tuple<Args...> from_json_tuple_impl_base(BasicJsonType&& j, index_sequence<
     return std::make_tuple(std::forward<BasicJsonType>(j).at(Idx).template get<Args>()...);
 }
 
+template<typename BasicJsonType>
+std::tuple<> from_json_tuple_impl_base(BasicJsonType& /*unused*/, index_sequence<> /*unused*/)
+{
+    return {};
```

Comment:
```suggestion
    return array_t();
```

Braced initialization is somewhat cursed in this library (see https://json.nlohmann.me/home/faq/#brace-initialization-yields-arrays). Please avoid it.

Reply:
This isn't returning `json`, it's returning `std::tuple<>`, so it would be `std::tuple<>{}` instead of `{}` if you still want that change.

Reply:
Oh sorry. Then never mind.